openc3/lib/openc3/conversions/object_write_conversion.rb

```diff
@@ -0,0 +1,42 @@
+# encoding: ascii-8bit
+
+# Copyright 2024 OpenC3, Inc.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# This file may also be used under the terms of a commercial license
+# if purchased from OpenC3, Inc.
+
+require 'openc3/conversions/object_read_conversion'
+
+module OpenC3
+  class ObjectWriteConversion < Conversion
```

Comment:
What is this for?

Reply:
Used with virtual, to write a hash of items, into a packet, and then fill in a BLOCK item.